doc/bluealsa.8.rst

```diff
@@ -1,105 +1,113 @@
-% BLUEALSA(8) $VERSION$ | System Manager's Manual
-%
-% August 2020
+========
+bluealsa
+========
 
-# NAME
+----------------------------
+Bluetooth Audio ALSA Backend
+----------------------------
 
-bluealsa - Bluetooth Audio ALSA Backend
+:Date: August 2020
+:Manual section: 8
+:Manual group: System Manager's Manual
+:Version: $VERSION$
 
-# SYNOPSIS
+SYNOPSIS
+========
 
 **bluealsa** [*OPTION*]...
 
-# DESCRIPTION
+DESCRIPTION
+===========
 
 **bluealsa** is a Linux daemon to give applications access to Bluetooth audio streams using the
 Bluetooth A2DP, HFP and/or HSP profiles.
 It provides a D-Bus API to applications, and can be used by ALSA applications via libasound plugins.
 
-# OPTIONS
+OPTIONS
+=======
 
-**-h**, **\--help**
-:   Output a usage message and exit.
+-h, --help
+    Output a usage message and exit.
 
-**-V**, **\--version**
-:   Output the version number and exit.
+-V, --version
+    Output the version number and exit.
 
-**-B**, **\--dbus**=*NAME*
-:   BlueALSA D-Bus service name suffix.
+-B, --dbus=NAME
```

Comment:
Add `NAME` after one letter option name - it will look better in the final man page. Same for `--device` and `--profile`.

Reply:
Yes, I somehow missed this one. Will change.